### DISCUSSION TITLE
Errors with running lint and validate, as mentioned in CONTRIBUTING.MD

### DISCUSSION THREAD

**Discussion by tarun360** [containers/podman#16403] [Q&A] [answered]
In `CONTRIBUTING.MD`, it's mentioned to run the following to do `make lint` and `make validate`
`podman build -t gate -f contrib/gate/Dockerfile .`
However there's no such file `contrib/gate/Dockerfile` in the repository.

Further it's also mentioned to do the following:
```
$ cd $HOME/devel/podman
$ podman run -it --rm -v $PWD:/usr/src/libpod:ro \
    --security-opt label=disable quay.io/libpod/gate:master \
    lint
```
This returns an error:
`ERROR: Expecting libpod repository root at /usr/src/libpod (gate container entrypoint)`

Also, why can't I just run `make lint` and `make validate`, instead of running these two additional things?

Lastly, If I run `make lint`, it doesn't pass and throws errors related to files which I haven't touched, for example:
```
pkg/domain/infra/tunnel/images.go:21:2: "github.com/containers/podman/v4/pkg/errorhandling" imported but not used (typecheck)
        "github.com/containers/podman/v4/pkg/errorhandling"
cmd/podman/containers/ps.go:359:51: l.StartedAt undefined (type psReporter has no field or method StartedAt) (typecheck)
                t := units.HumanDuration(time.Since(time.Unix(l.StartedAt, 0)))
```
      
(There were additonal errors, I just copy pasted two of them here)

**Comment by vrothberg** [comment]
Thanks for reaching out, @tarun360.  It looks like the CONTRIBUTING.MD is outdated.

@cevich WDYT?

**Comment by cevich** [comment, marked as answer]
Oh ya, woops, that's horribly outdated.  I'll file an issue.
